packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -236,8 +244,9 @@ class CupertinoSheetTransition extends StatefulWidget {
     Animation<double> animation,
     Animation<double> secondaryAnimation,
     bool allowSnapshotting,
-    Widget? child,
-  ) {
+    Widget? child, {
+    double? topGap,
```

Comment:
Since we won't use this transition if there is a topGap, then we can remove this and not handle it.